x-pack/plugins/watcher/public/models/execute_details/execute_details.js

```diff
@@ -47,9 +48,16 @@ export class ExecuteDetails {
 
   formatTime(timeUnit, timeValue) {
     const now = 'now';
+
     if (timeValue === 0) {
       return now;
     }
+
+    // milliseconds is the default in Date Math
+    if (timeUnit === TIME_UNITS.MILLISECOND) {
+      return timeValue;
+    }
```

Comment:
Could you help me understand why we're returning the `timeValue` if the `timeUnit` is in ms? From my naive perspective it seems like we should still be saying "Hey, execute this watch in 50 milliseconds", which would translate into a `triggered_time` of `now+50ms`.

EDIT: Did some digging into the [date math docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#date-math) and it looks like datemath doesn't support `ms`. So I'm guessing the ES engineers have updated the `triggered_time` option to support units in just `ms` as well, which are interpreted as "the number of milliseconds from now". Is that right? If so then I think it's worth providing some more details in the comment to make that super clear, even if it does mean being more verbose. :)

Reply:
Thanks for pointing this out. I did some further investigation into this. At first, yaron and I were under the impression that `ms` was the default, but I spoke with the ES team and got clarification. You're correct, date math does not support `ms` and the lowest time value that is supported is `seconds`. I updated the code and removed the milliseconds option. (In the old UI, this was just a plain text input so the bug did not exist.)